configuration/pih/petl/jobs/sql/extractions/pmtct_contacts.sql

```diff
@@ -0,0 +1,89 @@
+-- set @locale = 'en';  -- uncomment for testing
+
+select program_workflow_state_id into @patientPregnant from program_workflow_state where concept_id = concept_from_mapping('PIH','1755');
```

Comment:
This might not be good enough.  Multiple workflows and states (in different programs and/or workflows) can share the same concept.  You need to make sure you are getting back the state for the workflow you are interested in.  A few utility functions to get programs, workflows, states would be useful.